src/frontend/qt_sdl/main.cpp

```diff
@@ -1418,6 +1465,65 @@ void ScreenPanelGL::onScreenLayoutChanged()
     setupScreenLayout();
 }
 
+
+static bool FileExtensionInList(const QString& filename, const QStringList& extensions, Qt::CaseSensitivity cs = Qt::CaseInsensitive)
+{
+    return std::any_of(extensions.cbegin(), extensions.cend(), [&](const auto& ext) {
+        return filename.endsWith(ext, cs);
+    });
+}
+
+static bool MimeTypeInList(const QMimeType& mimetype, const QStringList& superTypeNames)
+{
+    return std::any_of(superTypeNames.cbegin(), superTypeNames.cend(), [&](const auto& superTypeName) {
+        return mimetype.inherits(superTypeName);
+    });
+}
+
+
+static bool NdsRomByExtension(const QString& filename)
+{
+    return FileExtensionInList(filename, NdsRomExtensions);
+}
+
+static bool GbaRomByExtension(const QString& filename)
+{
+    return FileExtensionInList(filename, GbaRomExtensions);
+}
+
+static bool SupportedArchiveByExtension(const QString& filename)
+{
+    return FileExtensionInList(filename, ArchiveExtensions);
+}
+
+
+static bool NdsRomByMimetype(const QMimeType& mimetype)
+{
+    return  MimeTypeInList(mimetype, NdsRomMimeTypes);
```

Comment:
unnecessary space?

Reply:
I’m not entirely sure what you mean. If you mean the additional newlines, I’ve added those to visually delineate the blocks of similar functions. I’ll remove them if you prefer. If you mean the function body then please give me an example of the correct code style.

Reply:
```c++
    return MimeTypeInList(mimetype, NdsRomMimeTypes);
```

Reply:
Fixed in ca75766c9c821b17661a05be2de88de0c3c16821.